### ISSUE TITLE
HDR indicator

### ISSUE THREAD

**Issue by marcelloinfoweb** [moneytoo/Player#506]
It could have information on the panel when playing the HDR format, whether HDR10, + or Dolby Vision.

A practical example is audio, we can switch, when available, to AAC, DD+, TRUEHD, DTS, etc.

It could have something similar for video, such as videos that have HDR or Dolby Vision in the same file.

**Comment by Alexxlml** [comment]
This feature would be great, since the only way I had to check Dolby Vision support, was to play a local copy of Cyberpunk Edgerunners that only has the DV layer without HDR or SDR support and this had no purple tints in playback like VLC or MX Player.

Perhaps something similar to what televisions currently do by showing notifications when the content played has DV, HDR or DA.

**Comment by zhuhang-jasper** [comment]
Great idea.
It can be shown every time the video plays (at beginning, and every time pause/play happens)

**Comment by trlef19** [comment]
Would be nice

**Comment by marcelloinfoweb** [comment]
The project must be abandoned, unfortunately!

**Comment by Pentaphon** [comment]
I agree. HDR Support is a highlight of this player and a visual indicator is very useful to have.
